mlflow/tracking/client.py

```diff
@@ -516,6 +516,29 @@ def register_prompt(
 
         validate_prompt_name(name)
 
+        if is_databricks_unity_catalog_uri(self._registry_uri):
+            try:
+                registry_client.create_prompt(
+                    name=name, description=commit_message, tags=tags or {}
+                )
+            except MlflowException as e:
+                if e.error_code == "ALREADY_EXISTS":
```

Comment:
Should we parse this into a databricks proto error code and compare that way? Seems like it might be more robust?